assets/js/footer.js

```diff
@@ -0,0 +1,9 @@
+function FIX_THIS_S___(){
```

Comment:
May be `fixFooter`?

Reply:
Oh that was a joke I forgot to change it to something clean. It's a good thing I censored myself in anticipation of not remembering to change it

Reply:
I'll merge once you make this a better name.